packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -335,7 +336,13 @@ function ViewGrid< Item >( {
 						'dataviews-no-results': ! isLoading,
 					} ) }
 				>
-					<p>{ isLoading ? <Spinner /> : __( 'No results' ) }</p>
+					{ isLoading ? (
+						<p>
```

Comment:
I was careful here to keep the `<p>` around the `<Spinner>`. But I'm not sure how useful it actually is. You can see the extra margin the `<p>` provides in storybook, but that's because the data views in storybook do not stretch vertically.

In practice, the loading states within Gutenberg are centred which makes the margin from the `<p>` useless.






| With `<p>` | Without `<p>` |
|-|-|
| ![before](https://github.com/user-attachments/assets/c9fc3e0e-7361-4d88-a982-9e7ad5b6a286) | ![after](https://github.com/user-attachments/assets/c5982348-af6a-4f29-84ad-a14bbd181a1f) |